docs/2.3. Usage - Testing IS-10 Authorization.md

```diff
@@ -1,12 +1,16 @@
 # Testing IS-10 Authorization
 
-When testing IS-10 / BCP-003-02 implementations, ensure that a user is registered with the Authorization Server with a
-username and password that corresponds with the `AUTH_USERNAME` and `AUTH_PASSWORD` config options in the
-`nmostesting/UserConfig.py` file. These values should be changed to sensible values before running the IS-10 tests
-and will be used as the Basic Authentication mechanism when contacting the Authorization Server.
-
-When testing dynamic client registration, the contents of `test_data/IS1001/register_client_request_data.json` will be used in
-the body of the request when registering the client, and should comply with [RFC7591](https://tools.ietf.org/html/rfc7591). When
-testing the authorization code grant, the means by which consent is given by the resource owner will be implementation-specific.
-The contents of the file `test_data/IS1001/authorization_request_data.json` will be used as the body of the request to the
-authorization endpoint. **Please edit both files** to comply with the implementation under test.
+When testing IS-10 / BCP-003-02 implementations, there are a number of configuration parameters which need to be set
+correctly to match your environment. `ENABLE_AUTH` should be set to `True`, then each of the parameters which begins
+`AUTH_` should be considered to ensure it is set correctly.
+
+Each of the specification test suites can be run in authorized mode. When this is enabled, every request which is made

```

Comment:
Does this test auth on IS-09?

Reply:
https://github.com/AMWA-TV/nmos-testing/pull/561/files#diff-c8e77bc7d664c66e3e8b8a0d310dfcf7R24

No it does not.

```suggestion
Each of the specification test suites can be run in authorized mode where applicable. When this is enabled, every request which is made

```